app.js

```diff
@@ -60,23 +75,202 @@ nameForm.addEventListener('submit', (event) => {
 
 // After modal
 function showButton() {
-    submitBtn.classList.add('show-button');
+    submitPlayerSelectionBtn.classList.add('show-button');
 }
 
+submitPlayerSelectionBtn.addEventListener('click', () => {
+    getComputerValue();
+});
+
 function getComputerValue() {  
-    switch(randomNumber) {
-        case 0:
-            appState.computerSelectedItem = paper;
-            computerElements.paperElement.classList.add('item-visible');
-            break;
-        case 1:
-            appState.computerSelectedItem = rock;
-            computerElements.rockElement.classList.add('item-visible')
-            break;
-        default:
-            appState.computerSelectedItem = scissors;
-            computerElements.scissorsElement.classList.add('item-visible');
-    }
+    disablingBtn()
+    loader.classList.add('item-visible');
+
+    setTimeout( () => {
+        loader.classList.remove('item-visible');
+
+        const randomNumber = getRandom();
+        switch(randomNumber) {
+            case 0:
+                appState.computerSelectedItem = paper;
```

Comment:
we could make a small refactor and prepare a functions for all the cases, but we will make it in separate PR.